src/common/__tests__/graphHelpers.test.js

```diff
@@ -1,7 +1,34 @@
-import { graphHelpers } from '../graphHelpers';
+import { graphHelpers, getGraphHeight, getTooltipDimensions, getTooltipFontSize } from '../graphHelpers';
+import { helpers } from '../helpers';
+
+const { breakpoints } = helpers;
 
 describe('GraphHelpers', () => {
   it('should have specific functions', () => {
     expect(graphHelpers).toMatchSnapshot('helpers');
   });
+  it('should match graph heights at all breakpoints', () => {
```

Comment:
optional: next round have to remember to expand and test the `convertGraphData` function

Reply:
future :)